File: apprefurbished/src/main/java/com/androidcollider/easyfin/faq/FAQPresenter.java
package com.androidcollider.easyfin.faq;

import androidx.annotation.Nullable;

/**
 * @author Ihor Bilous
 */

public class FAQPresenter implements FAQMVP.Presenter {

    @Nullable
    private FAQMVP.View view;
    private FAQMVP.Model model;


    public FAQPresenter(FAQMVP.Model model) {
        this.model = model;
    }

    @Override
    public void setView(@Nullable FAQMVP.View view) {
        this.view = view;
    }

    @Override
    public void loadInfo() {
        if (view != null) {
            view.setInfo(model.getInfo());
        }
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/faq/FAQPresenter.kt
package com.androidcollider.easyfin.faq

/**
 * @author Ihor Bilous
 */
class FAQPresenter(private val model: FAQMVP.Model) : FAQMVP.Presenter {
    private var view: FAQMVP.View? = null
    override fun setView(view: FAQMVP.View?) {
        this.view = view
    }

    override fun loadInfo() {
        view?.setInfo(model.info)
    }
}